BFWControls/Modules/Alert/View/AlertView.swift

```diff
@@ -91,7 +91,7 @@ import UIKit
     }
     
     open func button(forTitle title: String) -> UIButton? {
-        return actions.first { $0.title == title }?.button
+        return actions.first { $0.title?.uppercased() == title.uppercased() }?.button
```

Comment:
It's conceivable that alert buttons might be uppercase and lowercase versions of the same text, so ignoring the case here isn't a logical fit. Can you instead implement or override in your subclass?